src/app/pages/LendingPage/components/CurrencyContainer/UserLendingInfo.tsx

```diff
@@ -47,11 +50,36 @@ export const UserLendingInfo: React.FC<IUserLendingInfoProps> = ({
     account,
   );
 
+  const {
+    value: balanceOfCall,
+    loading: balanceOfLoading,
+  } = useLending_balanceOf(asset, account);
+
+  const {
+    value: checkpointPrice,
+    loading: checkpointLoading,
+  } = useLending_checkpointPrice(asset, account);
+
+  const {
+    value: tokenPrice,
+    loading: tokenPriceLoading,
+  } = useLending_tokenPrice(asset);
+
   const balance = useMemo(() => {
     return bignumber(balanceCall).minus(profitCall).toString();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [balanceCall, profitCall, asset]);
 
+  const totalProfit = useMemo(() => {
+    return bignumber(tokenPrice)
+      .sub(checkpointPrice)
+      .mul(balanceOfCall)
+      .div(10e18)
+      .add(profitCall)
+      .toString();
```

Comment:
If there is a chance that wei amount will have a decimals - make sure to remove them, otherwise dapp may crash because wei can be integers only.

```suggestion
      .toFixed(0);
```